### DISCUSSION TITLE
NavX2 AHRS Class Documentation

### DISCUSSION THREAD

**Discussion by benzemp** [wpilibsuite/2024Beta#11] [Q&A] [answered]
Is there a source for reference documentation for the NavX2 vendordep / AHRS class? I was able to find a javadoc jar on the maven the vendordep json links to, but I'm not sure it's up to date. An online version that isn't raw html similar to the WPILib, REV, and CTRE API docs would be helpful.

**Comment by Jm-Taylor** [comment, marked as answer]
Do you mean the doxygen-style docs here:

CPP (https://www.kauailabs.com/public_files/navx-mxp/apidocs/c++/)
Java (https://www.kauailabs.com/public_files/navx-mxp/apidocs/java/)

or are you talking a full-style docs, like RTD?
